lib-rca/src/main/java/com/facebook/openwifi/librca/models/LinkStats.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.models;
+
+import java.util.List;
+
+/** 
+ * Aggregation Statistics Model of InputStats. Aggregate by bssid, station and RadioConfig.
+ */
+public class LinkStats {
+    public static class RadioConfig {
```

Comment:
I think we should extract the `RadioConfig` class out separately and avoid duplicate code (it's the same as `InputStats.RadioConfig`

Reply:
Which one do you suggest? A separate file or just put it in the same file?

Reply:
We discussed offline